README.md

```diff
@@ -163,6 +163,46 @@ helm tiller run \
 -    --set container.image=eu.gcr.io/census-eq-ci/eq-survey-runner-benchmark:latest
 ```
 
+## Visualising Benchmark Results
+
+You can use the `visualise_results.py` script to visualise benchmark results over time. 
+
+### Pre-Requisites
+
+You need to be authenticated with GCP in order to download the benchmark results. To do this:
+
+Log In with gcloud auth using:
+
+`gcloud auth login`
+
+`gcloud auth application-default login`
+
+Alternatively you can use the project service Account:
+
+- Go to the `eq-daily-performance-test` project in GCP
+- Go to IAM&Admin and open up the Service Accounts tab
+- Click on the `results-analysis` service account
+- Click edit, then, click 'Create Key'
+- Select JSON and store the file in a safe location - the path for this file will be used as an environment variable for the `get_benchmark_results.py` script.
+
+### Download Benchmark Results
+
+Run the `get_benchmark_results.py` to download the latest benchmark results from GCP. This script requires two environment variables to be set:
+
+```
+GCS_OUTPUT_BUCKET - eq-daily-performance-test
+```
+```
+GOOGLE_APPLICATION_CREDENTIALS - <path_to_json_credentials_file> pipenv run python -m scripts.get_benchmark_results
+```
+
+### Run the Visualise Results script
+
+The `visualise_results.py` script will run against any benchmark results stored in an `outputs/daily-test` directory.
+
+A line chart will be generated and saved as `performance_graph.png`.
```

Comment:
This needs to be updated to reflect that the directory is now passed in. This should also have a run command and example specified. As the directory is passed in as an argument to the script here, we should consider doing the same for the bucket name in the get script above.